pkg/client/docker/docker.go

```diff
@@ -41,8 +42,10 @@ func New(opts Options, log *logrus.Entry) (*Client, error) {
 	}
 	retryclient.HTTPClient.Timeout = 10 * time.Second
 	retryclient.RetryMax = 10
-	retryclient.RetryWaitMax = 2 * time.Minute
+	retryclient.RetryWaitMax = 10 * time.Minute
 	retryclient.RetryWaitMin = 1 * time.Second
+	// This custom backoff will fail requests that have a max wait of the RetryWaitMax
+	retryclient.Backoff = util.HTTPBackOff
```

Comment:
Probably worth adding a note here that this could be removed if https://github.com/hashicorp/go-retryablehttp/issues/247 is addressed.